CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [0.13.0] - 2024-06-17
+
+### Added Changes
+- `getSDK` include a param to choose to instantiate the Prime SDK instead of the Modular SDK
+- Added Etherspot Modular SDK `installModule` and `uninstallModule` to hook `useEtherspotModules`
+- `EtherspotContextProvider` updated with choice of instantiatiating the Prime SDK or Modular SDK
+- Added `isModular` to context `EtherspotContextProvider`
+
+### Breaking Changes
+- Etherspot Modular SDK implemented to Transaction Kit as the default SDK
```

Comment:
```suggestion
- Etherspot Modular SDK implemented to Transaction Kit as the default `accountTemplate`
```

Reply:
Note the above will make sense after reading the comments below

Reply:
Absolutely!